tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3148,8 +3145,7 @@ TEST_F(MatmulSchedulerTest, HSH_TT) {
       matmul_cparams);
   auto cg_outputs = fe.runFusion({inputs.first, inputs.second});
   auto tref = atMatmul(inputs.first.squeeze(), inputs.second.squeeze(), layout);
-  EXPECT_TRUE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
-  */
+  EXPECT_FALSE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
```

Comment:
We could place a comment at these `EXPECT_FALSE` lines that these are disabled until the mma is scheduled.